lib/src/test/java/com/toedter/spring/hateoas/jsonapi/support/polymorphy/SuperEntity.java

```diff
@@ -24,5 +24,6 @@
         @JsonSubTypes.Type(value = SuperEChild.class, name = "superEChild"),
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
Hm, what do you mean by `Methods`? `name` would be the value of the type attribute. I don't see any method references in the highlighted code. Could you please explain, what you exactly mean? Thx